src/lib/y2partitioner/sequences/add_partition.rb

```diff
@@ -31,6 +33,7 @@ def initialize(disk_name)
         textdomain "storage"
         @disk_name = disk_name
         @ptemplate = PartitionTemplate.new
+        @options = FormatMount::Options.new
```

Comment:
Are the options really independent from the partition template? I hope so, but if not, then we have to combine them.
BTW, my mistake to have placed PartitionTemplate in the Sequences namespace.